app/views/content/landing/how-to-fund-your-teacher-training.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "How to fund your teacher training"
-description: Find out how you could fund your teacher training with bursaries and scholarships available up to £30k, depending on the subject you're training to teach.
+description: Find out how you could fund your teacher training with bursaries and scholarships available up to %{scholarships_generic_maxshortened}, depending on the subject you're training to teach.
```

Comment:
I think this might cause us a problem @MylesJarvis - it is not going to render properly in organic search results

<img width="1074" alt="image" src="https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/7fa950b5-ec08-4e21-97ac-0699e469ad73">

Reply:
Yep, you're right! I'll amend this one back and we'll have to keep an eye on it manually.